src/components/Search.jsx

```diff
@@ -123,7 +124,7 @@ export const Search = ({
 
     // Show dropdown only when `isOpen` is set to `true` and there are items in the list.
     const open = (isOpen || !canClose || shouldStayOpen) && itemList.length > 0 && !shouldHideMenu;
-    const noResultFound = open && !isLoading && itemList.length <= 1;
+    const noResultFound = open && !isLoading && itemList.length <= 1 && inputValue.length < minChars;
```

Comment:
Did we need to show noResultFound, even if the inout is less than minChars ?

Reply:
Hide 'No results found' when input value length is less then minChars